### DISCUSSION TITLE
废话杂谈

### DISCUSSION THREAD

**Discussion by Minemetero** [Minemetero/Table-Tennis-Counter#7] [General]
我们很需要这个，因为话太多了

**Comment by Minemetero** [comment]
6个issue够我消化1年了 ：）

**Reply by lingbopro** [reply]
啊

**Reply by lingbopro** [reply]
雀食

**Reply by Minemetero** [reply]
erm我当初说的话实现了

**Comment by Minemetero** [comment]
![image_2024-06-14_102738422](https://github.com/Minemetero/Table-Tennis-Counter/assets/156386015/8af14114-1c76-45c4-a912-6d7c636e90e4)
以JS为主的网页。。。

**Reply by lingbopro** [reply]
> ![image_2024-06-14_102738422](https://github.com/Minemetero/Table-Tennis-Counter/assets/156386015/8af14114-1c76-45c4-a912-6d7c636e90e4)
> 以JS为主的网页。。。

没毛病

**Comment by lingbopro** [comment]
《论我刚才打开VS结果刚弹出起始窗口就闪退了这件事》

**Reply by Minemetero** [reply]
听过去游戏闪退，你这个闪退第一次听

**Reply by lingbopro** [reply]
> 听过去游戏闪退，你这个闪退第一次听

我就这么一说（

**Comment by lingbopro** [comment]
Avalonia 好玩（（（

**Reply by Minemetero** [reply]
你送我一个网页呗 :)

**Reply by Minemetero** [reply]
MC 1.21 横空出世 ahhhhhhhhhhh

**Reply by lingbopro** [reply]
> 你送我一个网页呗 :)

这个...我正研究Avalonia呢

**Reply by lingbopro** [reply]
> MC 1.21 横空出世 ahhhhhhhhhhh

ohh

**Reply by Minemetero** [reply]
Dreamweave 是我见过最差的网页编辑平台

**Reply by lingbopro** [reply]
> Dreamweave 是我见过最差的网页编辑平台

啊？

**Reply by Minemetero** [reply]
你知道Dreamweaver嘛？

**Reply by Minemetero** [reply]
我见过最差的网页开发平台，全是过时的东西 :(

**Comment by lingbopro** [comment]
我整[Avalonia](https://avaloniaui.net/)的时候出了个离谱的bug，生成器生成出来的主窗口（`MainWindow`）代码报`CS0542`（`成员名称不能与它们的封闭类型相同`）错误，经过一番排查和头脑风暴以及必应搜索之后，毫无头绪...  
后来突然想到查找一下这个类名，发现在主窗口的`x:Name`属性中也用了这个名字，把它改掉就好了...  
最后总结出来一点，窗口组件的`x:Name`和类名不能相同。。。

真就改bug纯靠灵光一现啊

**Comment by Minemetero** [comment]
找到竞争对手了https://github.com/r48n34/Modern-Table-Tennis-Score-Board

**Reply by Minemetero** [reply]
虽然UI好看但是。。。我仔细一看发现用不了！不知道咋加分

**Reply by lingbopro** [reply]
> 虽然UI好看但是。。。我仔细一看发现用不了！不知道咋加分

《找到竞争对手了，但是竞争对手用不了》

**Reply by Minemetero** [reply]
UI方面我对比一圈就我们和他有得一拼，功能方面其实我自认为是我们最强

**Reply by lingbopro** [reply]
对手这个最大的问题是部署在Vercel，大陆裸连连不上

**Reply by lingbopro** [reply]
> 虽然UI好看但是。。。我仔细一看发现用不了！不知道咋加分

那个其实能用，不过卡片上下滑动的交互逻辑很不直观  

不过计时器的功能可以借鉴一下

**Reply by lingbopro** [reply]
> 虽然UI好看但是。。。我仔细一看发现用不了！不知道咋加分

并且那个的交互逻辑很难上手，我倒腾了一圈都没找到怎么用

**Reply by lingbopro** [reply]
> > 虽然UI好看但是。。。我仔细一看发现用不了！不知道咋加分
> 
> 并且那个的交互逻辑很难上手，我倒腾了一圈都没找到怎么用

研究了五分钟才发现点标题下面的小蓝按钮可以下一轮（

**Reply by lingbopro** [reply]
而且对手的用的 React，不如原版 HTML/CSS/JS 轻量，JSX/TSX 还得编译

**Reply by lingbopro** [reply]
emmm...

感觉我损对手损的有点过分呢...

**Reply by Minemetero** [reply]
那个人把issue给关了这导致我投诉不了他

**Reply by lingbopro** [reply]
> 那个人把issue给关了这导致我投诉不了他

啊？
什issue

**Comment by lingbopro** [comment]
事实证明兼容性很重要

我就换了个Minecraft字体，一堆软件的字体显示都不对

最过分的是这个日历
![Screenshot_20240622_170746.jpg](https://github.com/Minemetero/Table-Tennis-Counter/assets/148617407/c60011ff-c729-44ef-8d3a-b61ca8a6e682)

**Reply by Minemetero** [reply]
ahhhhhhh

**Comment by Minemetero** [comment]
![image_2024-06-22_190450336](https://github.com/Minemetero/Table-Tennis-Counter/assets/156386015/2baa3513-c143-4f52-8d23-04d3b8aa6056)
可以加一些这种可爱的小图标，也是竞争对手那看到的 :) 反正他用的MIT版权上没问题

**Reply by lingbopro** [reply]
我去看了一下，发现这个图标很像 Font Awesome 的一个图标啊

![Screenshot_20240623_100659_com.microsoft.emmx.jpg](https://github.com/Minemetero/Table-Tennis-Counter/assets/148617407/6e02c45f-c8c8-4b0d-af8f-cd065d1020d2)

虽然有点小区别，但是总体差不多

**Reply by Minemetero** [reply]
这是一模一样好吧

**Reply by lingbopro** [reply]
> 这是一模一样好吧

还是不太一样
那个相比 Font Awesome 的更圆润，而且方向不一样，球的位置也不同

**Reply by Minemetero** [reply]
Font Awesome是个好网站我之前都不知道

**Reply by lingbopro** [reply]
> Font Awesome是个好网站我之前都不知道

Font Awesome, Bootstrap Icons, Material Icons 这三个都是比较好的iconfont

我比较喜欢用 Bootstrap Icons 和 Material Icons

**Reply by r48n34** [reply]
那個 Icon 好像是用 https://tabler.io/icons 的，反正也是 icon大集成，有Font Awesome也不出奇

**Reply by Minemetero** [reply]
> 那個 Icon 好像是用 https://tabler.io/icons 的，反正也是 icon大集成，有Font Awesome也不出奇

新人!?新活人

**Reply by Minemetero** [reply]
> 那個 Icon 好像是用 https://tabler.io/icons 的，反正也是 icon大集成，有Font Awesome也不出奇

你的那个https://github.com/r48n34/Modern-Table-Tennis-Score-Board 能不能开个issue

**Reply by r48n34** [reply]
> > 那個 Icon 好像是用 https://tabler.io/icons 的，反正也是 icon大集成，有Font Awesome也不出奇
> 
> 你的那个https://github.com/r48n34/Modern-Table-Tennis-Score-Board 能不能开个issue

開了，不過我也很少用的，有空先來看一看

兄弟你這個也不錯阿

**Reply by lingbopro** [reply]
哇，新的活人

不过我更好奇的是这个小repo是怎么被发现的（（（

**Reply by Minemetero** [reply]
> 哇，新的活人
> 
> 不过我更好奇的是这个小repo是怎么被发现的（（（

我把他的repo的URL给贴了过来,顺着网线爬过来了。。。他就是我说的那个竞争对手。。。

**Comment by lingbopro** [comment]
![image](https://github.com/Minemetero/Table-Tennis-Counter/assets/148617407/5cb11146-3970-45f2-8119-acaccde72966)

这很有意思x2

**Reply by Minemetero** [reply]
这确实很有意思

**Reply by Minemetero** [reply]
梅开二度了

**Reply by lingbopro** [reply]
关键是这个网站是查看仓库的 Good First Issue 的，我们这仓库也没 Good First Issue 啊

**Reply by lingbopro** [reply]
还有一种可能，就是有人先从这个网站跳转到了GitHub，然后在 GitHub 阴差阳错的进入了这个 repo

但是这样的话referer不应该指向那个网站啊
回头我去看看MDN怎么说的

**Reply by Minemetero** [reply]
https://www.thyuu.com 好酷，好强

**Reply by Minemetero** [reply]
![image_2024-06-23_110857504](https://github.com/Minemetero/Table-Tennis-Counter/assets/156386015/07318ce3-853b-4242-a8e5-eece2eb765f8)
谁说没有Good First Issue？

**Reply by lingbopro** [reply]
> ![image_2024-06-23_110857504](https://github.com/Minemetero/Table-Tennis-Counter/assets/156386015/07318ce3-853b-4242-a8e5-eece2eb765f8)
> 谁说没有Good First Issue？

emmm
有道理🤔

**Reply by lingbopro** [reply]
> https://www.thyuu.com 好酷，好强

这网站挺好看啊

**Reply by Minemetero** [reply]
非常好看，想要。。。

**Reply by lingbopro** [reply]
> https://www.thyuu.com 好酷，好强

小站点特有的 [开往](https://www.travellings.cn/) (bushi

**Reply by Minemetero** [reply]
是这么回事

**Reply by Minemetero** [reply]
一个人做梦那就是个梦，一群人做梦是现实

**Reply by Minemetero** [reply]
我喜欢这个句子

**Reply by lingbopro** [reply]
> 是这么回事

啊？

**Reply by lingbopro** [reply]
> https://www.thyuu.com 好酷，好强

证实了，这个网站是基于WordPress的
看起来像是自己写的插件

<img width="620" alt="image" src="https://github.com/Minemetero/Table-Tennis-Counter/assets/148617407/21eb2f01-d344-4283-8e81-adeacb4b43cd">

**Comment by Minemetero** [comment]
![image_2024-06-23_111847623](https://github.com/Minemetero/Table-Tennis-Counter/assets/156386015/af3f0413-4bf4-4c54-9801-5a21e45ff39c)
你不觉得这个player放的位置有些别扭吗

**Reply by Minemetero** [reply]
是不是贴着上面那个框太近了啊

**Reply by lingbopro** [reply]
没事，改个margin的事，麻烦不了一点

**Comment by lingbopro** [comment]
emmm
这咋有个[还在排队的上周的pages部署任务](https://github.com/Minemetero/Table-Tennis-Counter/actions/runs/9508925104)

[![image](https://github.com/Minemetero/Table-Tennis-Counter/assets/148617407/c41ebaf0-31c4-4501-ad88-68b141b51d1d)](https://github.com/Minemetero/Table-Tennis-Counter/actions/runs/9508925104)

**Reply by lingbopro** [reply]
这个工作流我还停不了...

**Reply by Minemetero** [reply]
。。。。。

**Comment by lingbopro** [comment]
![image](https://github.com/Minemetero/Table-Tennis-Counter/assets/148617407/b87c52d4-16b9-428c-94d9-12cff21fb304)

啊？
24次clone？
16个人clone过？？？

**Reply by Minemetero** [reply]
克隆这东西干啥啊，线上都部署了

**Reply by Minemetero** [reply]
该概率是系统坏了 肯定

**Reply by lingbopro** [reply]
我的easy-github-hosts也显示有很多clone

**Reply by lingbopro** [reply]
话说
这个该不会把pull算进去了吧

**Comment by lingbopro** [comment]
我觉得可以开一个Project，用来规划发展方向`:)`

**Reply by lingbopro** [reply]
mmmm...

可以[给我一个Project修改权限](https://docs.github.com/zh/issues/planning-and-tracking-with-projects/managing-your-project/managing-access-to-your-projects#managing-access-for-user-level-projects)吗

**Comment by Minemetero** [comment]
我好像一直在摸鱼。。。

**Reply by lingbopro** [reply]
啊（（（（（

**Comment by Minemetero** [comment]
其实我觉得原来的界面简洁明了 :(

**Reply by lingbopro** [reply]
那我就有一个想法了

让用户可以切换新旧UI

**Reply by Minemetero** [reply]
但是我们没保留源代码吧，不然就创建新分支然后回滚

**Reply by Minemetero** [reply]
> 那我就有一个想法了
> 
> 让用户可以切换新旧UI

一个叫Morden一个叫Classic 聪明如我

**Reply by Minemetero** [reply]
整体框架不动改UI 嗯嗯很好

**Reply by lingbopro** [reply]
> 但是我们没保留源代码吧，不然就创建新分支然后回滚

可以直接从提交创建分支的吧

**Reply by lingbopro** [reply]
> > 那我就有一个想法了
> > 
> > 让用户可以切换新旧UI
> 
> 一个叫Morden一个叫Classic 聪明如我

但是有一个问题
GitHub Pages一个项目只能有一个分支部署上去

**Reply by Minemetero** [reply]
就放一起呗

**Reply by Minemetero** [reply]
一个叫index1 一个叫index要用的时候引导过去

**Reply by Minemetero** [reply]
简单

**Reply by lingbopro** [reply]
> 一个叫index1 一个叫index要用的时候引导过去

这文件名不好啊
应该一个叫index一个叫legacy或者classic

**Reply by lingbopro** [reply]
> 就放一起呗

所以说我们可以新建一个文件夹（比如叫legacy），然后把旧版的东西都放进去

**Reply by Minemetero** [reply]
听上去很不错

**Reply by Minemetero** [reply]
不过我觉得我们应该只是用一个html的文件，然后对现在有的东西进行重组。也就是说只切换布局，其他不变

**Reply by lingbopro** [reply]
> 不过我觉得我们应该只是用一个html的文件，然后对现在有的东西进行重组。也就是说只切换布局，其他不变

你的意思是说只使用旧版的UI结构，组件库还用新版的sober？

**Reply by Minemetero** [reply]
en

**Comment by Minemetero** [comment]
![image_2024-06-25_112004079](https://github.com/Minemetero/Table-Tennis-Counter/assets/156386015/f6a7059f-3044-4b34-ae1d-a790cb320905)
不错打出了高分

**Reply by Minemetero** [reply]
![image_2024-06-25_112541162](https://github.com/Minemetero/Table-Tennis-Counter/assets/156386015/9da455d9-b9d3-4881-bfb1-f0a3a061d2d8)
差异都好大啊

**Reply by lingbopro** [reply]
mmmm

**Comment by lingbopro** [comment]
突然sober获取不上了，访问啥都是找不到DNS地址，结果排查了一圈之后发现是Edge的DNS服务商连不上...
虚惊一场

![image](https://github.com/Minemetero/Table-Tennis-Counter/assets/148617407/e1131ca9-fa62-4564-9d5f-7f54f201f938)

**Reply by Minemetero** [reply]
我觉得我们需要把sober写进我们的README.md作为感谢

**Reply by lingbopro** [reply]
> 我觉得我们需要把sober写进我们的README.md作为感谢

那当然
毕竟sober可是撑起了这个repo的UI的半壁江山啊

回头我顺便加上吧

**Reply by Minemetero** [reply]
你更新迭代太快了，我脑袋没跟上

**Reply by lingbopro** [reply]
> 你更新迭代太快了，我脑袋没跟上

呃啊呃啊呃啊

**Comment by Minemetero** [comment]
我准备用我的新更新来冲击你的大脑

**Reply by lingbopro** [reply]
啊？

**Reply by Minemetero** [reply]
我放弃了我跑路了，我搞你的Github-Hosts算了 （ 这个网页上升到了和我不匹配的高度（

**Reply by lingbopro** [reply]
> 我放弃了我跑路了，我搞你的Github-Hosts算了 （ 这个网页上升到了和我不匹配的高度（

mmmm  
那我也去搞别的了

**Reply by lingbopro** [reply]
我打算给sober做一个VSCode扩展  

~~简而言之就是抄隔壁MDUI（~~

**Reply by Minemetero** [reply]
> > 我放弃了我跑路了，我搞你的Github-Hosts算了 （ 这个网页上升到了和我不匹配的高度（
> 
> mmmm 那我也去搞别的了

啥，你先把Serve的Bug给我修好再跑

**Reply by Minemetero** [reply]
> 我打算给sober做一个VSCode扩展
> 
> ~简而言之就是抄隔壁MDUI（~

哈？

**Reply by Minemetero** [reply]
> > 我放弃了我跑路了，我搞你的Github-Hosts算了 （ 这个网页上升到了和我不匹配的高度（
> 
> mmmm 那我也去搞别的了

你别挖个坑让我填啊

**Reply by lingbopro** [reply]
> > > 我放弃了我跑路了，我搞你的Github-Hosts算了 （ 这个网页上升到了和我不匹配的高度（
> > 
> > 
> > mmmm 那我也去搞别的了
> 
> 啥，你先把Serve的Bug给我修好再跑

这个这个，我没复现啊

**Reply by Minemetero** [reply]
什么玩意

**Comment by Minemetero** [comment]
UI 还得改 info显示比赛顺序但这很重要每次切换Tab太麻烦了，最好把比赛顺序改成实时显示这一组结束了显示下一组

**Comment by Minemetero** [comment]
![image_2024-07-04_164208952](https://github.com/Minemetero/Table-Tennis-Counter/assets/156386015/67fe219f-31ad-4e36-aa24-6eca991b1701)
Sober 噶了？

**Reply by lingbopro** [reply]
um？

我不到啊

**Reply by lingbopro** [reply]
不如提个issue问问

**Comment by lingbopro** [comment]
话说，这个repo多久没更新了

**Reply by Minemetero** [reply]
额，要干的功能都摆在那里可以干了 :)

**Comment by Minemetero** [comment]
https://github.com/JohannesWiller-ai/TableTennis
这个功能我也想要qwq

**Comment by Minemetero** [comment]
![image_2024-10-26_113158771](https://github.com/user-attachments/assets/724cbfb4-39f2-43ec-ad92-c874eecf1495)
打肿脸充胖子的我，issue塞满自产自销

**Comment by Minemetero** [comment]
![image_2025-02-21_172124005](https://github.com/user-attachments/assets/5144cdf8-6a2e-4d08-a0be-4cae8cd466ea)
真奇怪。。。谁天天克隆这个repo...

**Reply by lingbopro** [reply]
反正不是我干的（

**Comment by Minemetero** [comment]
<img width="154" alt="{EE80E5BA-C8E6-4812-B484-B816C00C9FBF}" src="https://github.com/user-attachments/assets/23f28080-2066-419e-973a-f2f9830e63d0" />

这两文件的内容好像是一样的，vite是怎么运转的

**Reply by lingbopro** [reply]
> <img alt="{EE80E5BA-C8E6-4812-B484-B816C00C9FBF}" width="154" src="https://private-user-images.githubusercontent.com/156386015/445402540-23f28080-2066-419e-973a-f2f9830e63d0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDc3MzIyNTksIm5iZiI6MTc0NzczMTk1OSwicGF0aCI6Ii8xNTYzODYwMTUvNDQ1NDAyNTQwLTIzZjI4MDgwLTIwNjYtNDE5ZS05NzNhLWYyZjk4MzBlNjNkMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUyMFQwOTA1NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZmRjNTdmYmJmYmYzMGFhMWZlNWRkNDkwMGI4YzI3MWJhMjM5ZTU3M2Y3MDEwNmI1NTAyZGJiNTE1Nzk2MGEyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.m3Oaw-DiYqmyJxEBdd4uXBPYu-BYxpOAEELHLDKIMgo">
> 这两文件的内容好像是一样的，vite是怎么运转的

不一样啊，map那个是源映射

**Reply by Minemetero** [reply]
> > <img alt="{EE80E5BA-C8E6-4812-B484-B816C00C9FBF}" width="154" src="https://private-user-images.githubusercontent.com/156386015/445402540-23f28080-2066-419e-973a-f2f9830e63d0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDc3MzIyNTksIm5iZiI6MTc0NzczMTk1OSwicGF0aCI6Ii8xNTYzODYwMTUvNDQ1NDAyNTQwLTIzZjI4MDgwLTIwNjYtNDE5ZS05NzNhLWYyZjk4MzBlNjNkMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUyMFQwOTA1NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZmRjNTdmYmJmYmYzMGFhMWZlNWRkNDkwMGI4YzI3MWJhMjM5ZTU3M2Y3MDEwNmI1NTAyZGJiNTE1Nzk2MGEyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.m3Oaw-DiYqmyJxEBdd4uXBPYu-BYxpOAEELHLDKIMgo">
> > 这两文件的内容好像是一样的，vite是怎么运转的
> 
> 不一样啊，map那个是源映射

欸，活得lingbopro又安详地离开了

**Comment by Minemetero** [comment]
<img width="684" alt="{28412ED7-4E1B-4C19-AC76-5115380F00BB}" src="https://github.com/user-attachments/assets/922a7601-6ebc-4973-81ac-3e6e23106a5c" />

这个很不合理

**Reply by lingbopro** [reply]
《33 Unique Cloners》

除了把Actions也算进去了之外我找不到任何合理的解释…

**Reply by Minemetero** [reply]
是的。。。

**Comment by Minemetero** [comment]
<img width="53" height="32" alt="{514B9E63-87FC-471E-A042-32699E7669E6}" src="https://github.com/user-attachments/assets/3c35fdcb-4afa-4212-87d2-47a45265fe49" />

这个1.1.9是什么啊

**Reply by Minemetero** [reply]
https://www.npmjs.com/package/sober?activeTab=versions

**Reply by Minemetero** [reply]
https://www.npmjs.com/package/sober/v/2.0.0-alpha.2

这个有点东西哈
